kwalitee/config.yml

```diff
@@ -30,11 +28,7 @@ repolist:
   - url: https://github.com/OCR-D/ocrd_anybaseocr
     official: true
     compliant_cli: false
-  - url: https://github.com/ocr-d-modul-2-segmentierung/ocrd_pc_segmentation
```

Comment:
For the record, the repo is at https://github.com/ocr-d-modul-2-segmentierung/ocrd-pixelclassifier-segmentation now. But it's indeed disabled in ocrd_all as well and not required here.